docs/ux_design.md

```diff
@@ -44,12 +44,10 @@ These guidelines cover the main things you need to know about designing a great
 ### Deploying your Extension
 
 | ---------| ------- | 
-|[Submitting your Extension to the Extension Gallery]({{site.baseurl}}/docs/ux_extension_gallery.html) | How to style your extension in accordance with the Tableau brand and your personal/company brand.|
+|[Submitting your Extension to the Tableau Exchange]({{site.baseurl}}/docs/ux_extension_gallery.html) | How to style your extension in accordance with the Tableau brand and your personal/company brand.|
```

Comment:
I don't know the styleguide is it: "the Tableau Exchange" or "Tableau Exchange"

Reply:
I think we've mostly been using "the Tableau Exchange" from the earlier edit pass @d45 made. In my ear at least, adding "the" seems to make it sound more correct :)

Reply:
👍

Reply:
The Tableau Exchange